shapeshift/web
# feat: enable covalent jaypegs

## Description

Because why not, best case scenario we get some more data in, worst case scenario there's no user facing diff since failures are gracefully handled.

## Pull Request Type


## Issue (if applicable)

N/A

## Risk



None

## Testing



- Click the NFT tab as willywonka
- Ensure your Polygon Jaypegs are displayed (see the top-right card icon for the NFT icon), and address/opensea links are working. Note we're unable to get floor price from Covalent

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)

<img width="1358" alt="image" src="https://github.com/shapeshift/web/assets/17035424/402b73b4-3404-4826-9a51-1465adb3274d">
<img width="966" alt="image" src="https://github.com/shapeshift/web/assets/17035424/ebbf8d3a-0c10-445c-a4e0-b8030b6fee32">
